x/wasm/keeper/keeper.go

```diff
@@ -35,6 +35,9 @@ const DefaultInstanceCost uint64 = 40_000
 // DefaultCompileCost is how much SDK gas we charge *per byte* for compiling WASM code.
 const DefaultCompileCost uint64 = 2
 
+// DefaultEventAttributeCost is how much SDK gas we charge *per byte* for attribute data in events
+const DefaultEventAttributeCost uint64 = 10
```

Comment:
If they charge 2gas/byte to write to iavl storage, we should charge less to write to the event system.
iavl storage is more critical to keep small imo.

Maybe 1 instead of 10?